assets/notifications.js

```
@@ -0,0 +1,319 @@
+        
+        // Handle notification click
+        async handleNotificationClick(notification) {
+            if (!notification.read) {
+                await this.markAsRead(notification.id);
+            }
+            
+            // Close dropdown
+            this.dropdownOpen = false;
+            
+            // Navigate based on notification type
+            if (notification.metadata && notification.metadata.link) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)